src/applications/Machine.java

```diff
@@ -0,0 +1,53 @@
+package applications;
+
+import dataStructures.LinkedQueue;
+
+class Machine {
+    // data members
+    private LinkedQueue jobQ; // queue of waiting jobs for this machine
+    private int changeTime; // machine change-over time
+    private int totalWait; // total delay at this machine
+    private int numTasks; // number of tasks processed on this machine
+    private Job activeJob; // job currently active on this machine
+
+    // constructor
+    Machine() {
+        jobQ = new LinkedQueue();
+    }
+
+    public LinkedQueue getJobQ() {
+        return jobQ;
+    }
+
+    public int getChangeTime() {
+        return changeTime;
+    }
+
+    public void setChangeTime(int changeTime) {
```

Comment:
It seems that maybe this should be sent in as an argument to the constructor instead of using a setter like this.